sre-agent-xpander.ai/sre_agent.py

```
@@ -0,0 +1,108 @@
+            "Always run commands to fetch live data."
+        ]
+        return Agent(
+            model=OpenAIChat(id="gpt-4o"),
+            tools=[ThinkingTools(add_instructions=True), self.ktool],
+            instructions=instr,
+            storage=self.adapter.storage,
+            markdown=True,
+            add_history_to_messages=True
+        )
+
+async def _cli() -> None:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)